### PR TITLE
[DataFactory]Add AzurePostgreSql Sink and Oracle/SapTable/Netezza Partitions and Several Relational sets

### PR DESCRIPTION
**Pull request by wenbof-zz**
…titoins and dedicated datasets for relational

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by wenbof** [comment]
The local SDK building and testing looks good. Details can be referred from https://github.com/wenbof/azure-sdk-for-net/tree/b0715

**Comment by wenbof** [comment]
@praries880 Ping... could you please review? thanks.

**Comment by praries880** [comment]
@KrisBash can you kindly review this?

**Comment by wenbof** [comment]
> ### SDK Automation [[Logs](https://openapistorageprod.blob.core.windows.net/sdkautomation/prod/Azure/azure-rest-api-specs/6617/1/logs.txt)] (Generated from [8b1eb70](https://github.com/Azure/azure-rest-api-specs/pull/6617/commits/8b1eb70aa7fb73433c90ee6ebd12a8681fa2e4c6), Iteration 1)
> Failed Python: [Azure/azure-sdk-for-python](https://github.com/Azure/azure-sdk-for-python) [[Logs](https://openapistorageprod.blob.core.windows.net/sdkautomation/prod/Azure/azure-rest-api-specs/6617/1/Azure/azure-sdk-for-python/logs.txt)]
> Failed Java: [Azure/azure-sdk-for-java](https://github.com/Azure/azure-sdk-for-java) [[Logs](https://openapistorageprod.blob.core.windows.net/sdkautomation/prod/Azure/azure-rest-api-specs/6617/1/Azure/azure-sdk-for-java/logs.txt)]
> Failed Go: [Azure/azure-sdk-for-go](https://github.com/Azure/azure-sdk-for-go) [[Logs](https://openapistorageprod.blob.core.windows.net/sdkautomation/prod/Azure/azure-rest-api-specs/6617/1/Azure/azure-sdk-for-go/logs.txt)]
> Failed JavaScript: [Azure/azure-sdk-for-js](https://github.com/Azure/azure-sdk-for-js) [[Logs](https://openapistorageprod.blob.core.windows.net/sdkautomation/prod/Azure/azure-rest-api-specs/6617/1/Azure/azure-sdk-for-js/logs.txt)]

I executed AUTOREST locally while it seems there is no error. the targeted line 1489:4, 1472:4 looks good as well. I also built local SDK which can be built successfully. if there are any changes needed from my side, could you please advise? thanks

Line 1472 "SybaseTableDataset": {
      "x-ms-discriminator-value": "SybaseTable",
      "description": "The Sybase table dataset.",
      "type": "object",
      "allOf": [
        {
          "$ref": "#/definitions/Dataset"
        }
      ],
      "properties": {
        "typeProperties": {
          "description": "Sybase table dataset properties.",
          "x-ms-client-flatten": true,
          "$ref": "#/definitions/SybaseTableDatasetTypeProperties"
        }
      }
    },
Line 1489    "SybaseTableDatasetTypeProperties": {
      "description": "Sybase table dataset properties.",
      "properties": {

**Comment by wenbof** [comment]
+@hvermis

**Comment by wenbof** [comment]
@hvermis , @praries880 , @KrisBash Ping... could you please kindly review and merge? thanks.

**Comment by hvermis** [comment]
@KrisBash Waiting for you review to unblock our SDK release, please help.
This is very similar to other one you reviewed couple days ago.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json
Comment (hvermis) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: Remove this?

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json
Comment (hvermis) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: This too?

Comment (wenbof) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: this is a new setting that is ready for use. I would prefer to keep it in this change.  any specific concerns?

Comment (wenbof) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: by the way, this change has nothing to do with the dataset changes.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json
Comment (wenbof) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: Removed this as discussed. will add in another PR coming together with SapBwCube Dataset change.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json
Comment (hvermis) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: Oracle - upper case

**Review by hvermis — APPROVED** [review]

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json
Comment (wenbof) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]: Thanks. Updated.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json
Comment (praries880) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json]: is this naming intentional?
Shouldn't it be `postgresSql` vs `postgreSql`?

**Review by praries880 — APPROVED** [review]

**Review by KrisBash — APPROVED** [review]
LGTM

**Pull request merged**
